presto-raptor/src/main/java/com/facebook/presto/raptor/RaptorMetadata.java

```diff
@@ -472,6 +473,27 @@ public void renameColumn(ConnectorSession session, ConnectorTableHandle tableHan
     }
 
     @Override
+    public void dropColumn(ConnectorSession session, ConnectorTableHandle tableHandle, ColumnHandle column)
+    {
+        RaptorTableHandle table = (RaptorTableHandle) tableHandle;
+        RaptorColumnHandle raptorColumn = (RaptorColumnHandle) column;
+
+        List<TableColumn> existingColumns = dao.listTableColumns(table.getSchemaName(), table.getTableName());
```

Comment:
Also validate that this is not a bucket column or the temporal column. Maybe also disallow dropping sorting columns?